dashboards/lodestar_block_processor.json

```diff
@@ -3537,7 +3537,7 @@
       "collapsed": false,
       "datasource": {
         "type": "prometheus",
-        "uid": "prometheus_local"
+        "uid": "${DS_PROMETHEUS}"
```

Comment:
This was not caught by the dashboard linter? Could you take a look @nflaig

Reply:
yeah something is off with the linter, I will take a look at that.

Reply:
Fixed in https://github.com/ChainSafe/lodestar/pull/5221